spec/solidus/gateway/braintree_gateway_spec.rb

```diff
@@ -348,6 +348,8 @@
             it 'sends a bill address' do
               expected_params = {
                 billing: {
+                  first_name: address.first_name,
+                  last_name: address.last_name,
```

Comment:
I added first and last name to the expected params. The assertion at the end of the test is comparing to this expected_params. Is there more of an assertion you're looking for?